py/makeqstrdata.py

```diff
@@ -286,6 +286,13 @@
     "__get__",
     "__set__",
     "__delete__",
+    # From py/scope.c: used in `scope_simple_name_table` table.
+    "<module>",
```

Comment:
I'm not sure this should be added, because it's already in `static_qstr_list`.  And so it may be possible for there to be a duplicate definition of it as QDEF0?

Also, operators from `py/objtype.c` that are in `static_qstr_list` are not listed in this set, so for consistency `<module>` shouldn't be.

Reply:
Updated.